src/main/windows/callsWidgetWindow.ts

```diff
@@ -82,6 +82,15 @@ export default class CallsWidgetWindow extends EventEmitter {
         ipcMain.on(CALLS_WIDGET_SHARE_SCREEN, this.onShareScreen);
         ipcMain.on(CALLS_JOINED_CALL, this.onJoinedCall);
 
+        this.win.webContents.setWindowOpenHandler(() => {
+            return {
+                action: 'allow',
+                overrideBrowserWindowOptions: {
+                    autoHideMenuBar: true,
```

Comment:
Does autoHide imply that you can trigger the menu somehow? If so, can we prevent that? Maybe Devin knows?

Reply:
Sorry, forgot to tag you @devinbinnie

Reply:
According to the documentation it can be open by pressing the `Alt` key but I tested this to make sure it didn't interfere with keyboard shortcuts and couldn't get it to show, probably because we are overriding it already.

Reply:
I know on Linux sometimes the menu needs to be specifically removed. @streamer45 Did you test on Linux as well?

Reply:
@devinbinnie I did, although on Linux the single `ALT` key works (shows the menu which by default is hidden) but shortcuts work as well. The only OS I couldn't try would be Windows since I don't have a machine with that.

If we could specify `isMenuBarVisible` from within `overrideBrowserWindowOptions` it may be best but looking at documentation I am not sure it's possible. I guess as long as the menu hidden by default it shouldn't be a huge deal but let me know.

Reply:
@streamer45 This is the fix I did for Linux on the main window, you might want to do the same thing for the calls widget: https://github.com/mattermost/desktop/blob/master/src/main/windows/mainWindow.ts#L224

Windows I think should be fine, but I can quickly test it if you'd like.

Reply:
I am just not sure we can do it, we don't really have direct access to the window. I could do it on the parent but I am not sure that would transfer.

Reply:
@streamer45 Not sure what you mean by not having direct access to the window, you create the window here: https://github.com/mattermost/desktop/blob/47edeea60157b420da146af5ca49a79986bc5096/src/main/windows/callsWidgetWindow.ts#L60

Can we not add the same listener above?

Reply:
@devinbinnie That's the parent window though. The window we are targeting here is opened through JS code at https://github.com/mattermost/mattermost-plugin-calls/blob/2bea09af46ba1b370dbfa92106f969297abc0d50/webapp/src/components/call_widget/component.tsx#L1418-L1422

Reply:
Ahh gotcha, that's probably a bit of a problem then. If you plan on opening a window in Electron that isn't part of the popout stuff (shown here: https://github.com/mattermost/desktop/blob/47edeea60157b420da146af5ca49a79986bc5096/src/main/views/webContentEvents.ts#L181) then you might want to think about passing that control to Electron instead of trying to do it as part of the plugin itself.

That use case isn't really supported right now because it allows us to open arbitrary Electron windows which we want to avoid.

Reply:
Thanks @devinbinnie, yeah it's a bit tricky because this popout actually relies on the `window.opener` to point to the widget window since the call client and all connectivity live there. 

Given all this should be temporary as we are planning to get rid of this extra window on Desktop I am proposing we disable the `Alt+` bindings in the popup from the plugin side when running on desktop to avoid the menu to show up on Linux but otherwise proceed with the changes here. Would that work?

Reply:
Yeah I think that should be fine, go ahead and make those changes.

Reply:
@devinbinnie Changes on plugin side have been merged in https://github.com/mattermost/mattermost-plugin-calls/commit/982bb11f54ab5fa95093a6856c360caee24de4d4 so we should be good once this PR gets in.